lib/helpers/android.ts

```diff
@@ -814,45 +814,51 @@ const AndroidHelpers: AndroidHelpers = {
         `${SETTINGS_HELPER_PKG_ID} is already running. ` +
```

Comment:
`if (await adb.processExists(SETTINGS_HELPER_PKG_ID)) {` must be foreground process existing check as https://github.com/appium/appium-adb/pull/715